__docs__/wonder-blocks-dropdown/multi-select.accessibility.mdx

```diff
@@ -44,3 +44,18 @@ receives focus. This can be useful when the options contain icons or other infor
 that would need to be omitted from the visible label.
 
 <Canvas of={MultiSelectAccessibilityStories.UsingOpenerAriaLabel} />
+
+## Automatic screen reader announcements in `MultiSelect`
```

Comment:
Would it be helpful to document the specific Safari/VoiceOver issues and the related Safari bug tickets? So we know that's the existing behaviour and it is to be expected! (Same comment for SingleSelect)

Reply:
I added some notes!